config/private_sphinx.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN X25519 PRIVATE KEY-----
+daRMtv8iyVmZyA3DW+inNUKOjCTk8eAiF5hvYX9mMws=
```

Comment:
The secrets should come from env vars at deployment type, same as we always do, we should not have them on version control.

Reply:
keys are in fact injected as secrets, but I need to have some default set untill Im finished. will remove before publishing.
https://github.com/w3f/nym-mixnode-chart/blob/init/charts/nym-mixnode/templates/deployment.yaml#L44_L55